src/main/java/org/jenkinsci/plugins/scriptler/ScriptlerManagement.java

```diff
@@ -44,57 +57,58 @@
 import org.jenkinsci.plugins.scriptler.share.ScriptInfoCatalog;
 import org.jenkinsci.plugins.scriptler.util.ScriptHelper;
 import org.jenkinsci.plugins.scriptler.util.UIHelper;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
 import org.kohsuke.stapler.*;
 import org.kohsuke.stapler.interceptor.RequirePOST;
 
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServletResponse;
-import java.io.File;
-import java.io.IOException;
-import java.text.SimpleDateFormat;
-import java.util.*;
-import java.util.logging.Level;
-import java.util.logging.Logger;
-
 /**
  * Creates the link on the "manage Jenkins" page and handles all the web requests.
- * 
+ *
  * @author Dominik Bartholdi (imod)
  */
 @Extension
 public class ScriptlerManagement extends ManagementLink implements RootAction {
 
-    private final static Logger LOGGER = Logger.getLogger(ScriptlerManagement.class.getName());
-    private final static String MASTER = "(master)";
-    private final static String ALL = "(all)";
-    private final static String ALL_SLAVES = "(all slaves)";
+    private static final Logger LOGGER = Logger.getLogger(ScriptlerManagement.class.getName());
+    private static final String INVALID_PATH = "Invalid file path received: ";
+    private static final String INDEX = "index";
+    private static final String NOT_APPROVED_YET = "notApprovedYet";
+    private static final String CAN_BYPASS_APPROVAL = "canByPassScriptApproval";
+    private static final String SCRIPT = "script";
+    private static final String MASTER = "(master)";
+    private static final String CONTROLLER = "(controller)";
```

Comment:
Per https://github.com/jenkinsci/jenkins/pull/5425 should this not be "(built-in)"?

Reply:
@basil, #128 is opened to make this change. Do you think you'd be able to review it?